server/controllers/messages/getChannels.js

```diff
@@ -0,0 +1,7 @@
+const { messages: { getChannels } } = require('../../database/queries');
+
+exports.getChannels = (req, res, next) => {
+  getChannels()
+    .then(({ rows }) => res.send({ data: rows }))
```

Comment:
we agreed to use `res.json` i think 
```suggestion
    .then(({ rows }) => res.json({ isSuccess: true,  data: rows }))
```